src/app/agents/agents-routing.module.ts

```diff
@@ -41,6 +42,13 @@ const routes: Routes = [
             breadcrumb: 'Edit Agent'
         },
         canActivate: [AuthGuard,AgentGuard]},
+      {
+        path: 'benchmarks', component: BenchmarkComponent,
+        data: {
+          kind: 'edit-agent',
+          breadcrumb: 'Edit Agent'
+      },
+      canActivate: [AuthGuard,AgentGuard]},     
       ]
```

Comment:
The code has change, so pull the new code and change should be sth like this.

        {
          path: 'benchmarks', component: BenchmarkComponent,
          data: {
            kind: 'benchmark',
            breadcrumb: 'Benchmark'
        },
        canActivate: [IsAuth,CheckPerm]},